contingency/contingency-dsl/src/test/java/com/powsybl/contingency/dsl/GroovyContingencyListTest.java

```diff
@@ -156,4 +164,27 @@ void testExtension() throws IOException {
         assertEquals(1, contingencies.get(0).getExtensions().size());
         assertEquals("myTs", contingencies.get(0).getExtension(ProbabilityContingencyExtension.class).getProbabilityTimeSeriesRef());
     }
+
+    @ParameterizedTest
+    @Timeout(3)
+    @ValueSource(booleans = {false, true})
+    void testCancelTask(boolean isDelayed) throws Exception {
+
+        // Counters
+        waitForStart = new CountDownLatch(1);
+        waitForFinish = new CountDownLatch(1);
+        waitForInterruption = new CountDownLatch(1);
+
+        // Booleans
+        config = new AtomicBoolean(false);
+        interrupted = new AtomicBoolean(false);
```

Comment:
Why these values are not initialized directly in `testCancelShortTask`?

Reply:
no reason, it could (I just changed it)